fr/auth.json

```diff
@@ -246,5 +246,11 @@
       "email_greeting": "Salut,",
       "email_disclaimer": "Vous avez reçu cet email car vous avez demandé un code de confirmation de ${business_name}"
     }
```

Comment:
@funkycharlie currently the JSON breaks the validator due to a missing comma here

Reply:
I think that was already there before, I haven't touched those parts. But yeah that's all good